geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/udf/GeomesaPysparkFunctions.scala

```diff
@@ -0,0 +1,109 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark.jts.udf
+
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.functions.udf
+import org.locationtech.geomesa.spark.jts.udf.GeometricAccessorFunctions._
+import org.locationtech.geomesa.spark.jts.udf.GeometricCastFunctions._
+import org.locationtech.geomesa.spark.jts.udf.GeometricConstructorFunctions._
+import org.locationtech.geomesa.spark.jts.udf.GeometricOutputFunctions._
+import org.locationtech.geomesa.spark.jts.udf.GeometricProcessingFunctions.{ST_BufferPoint, ST_antimeridianSafeGeom}
+import org.locationtech.geomesa.spark.jts.udf.SpatialRelationFunctions._
+
+/**
+ * Re-wrapping the UDFs so we can access them from PySpark without using the SQL API.
+ */
+object GeomesaPysparkFunctions {
+
+  /* Geometric Accessor Functions */
+  def st_boundary: UserDefinedFunction = udf(ST_Boundary)
+  def st_coordDim: UserDefinedFunction = udf(ST_CoordDim)
+  def st_dimension: UserDefinedFunction = udf(ST_Dimension)
+  def st_envelope: UserDefinedFunction = udf(ST_Envelope)
+  def st_exteriorRing: UserDefinedFunction = udf(ST_ExteriorRing)
+  def st_geometryN: UserDefinedFunction = udf(ST_GeometryN)
+  def st_geometryType: UserDefinedFunction = udf(ST_GeometryType)
+  def st_interiorRingN: UserDefinedFunction = udf(ST_InteriorRingN)
+  def st_isClosed: UserDefinedFunction = udf(ST_IsClosed)
+  def st_isCollection: UserDefinedFunction = udf(ST_IsCollection)
+  def st_isEmpty: UserDefinedFunction = udf(ST_IsEmpty)
+  def st_isRing: UserDefinedFunction = udf(ST_IsRing)
+  def st_isSimple: UserDefinedFunction = udf(ST_IsSimple)
+  def st_isValid: UserDefinedFunction = udf(ST_IsValid)
+  def st_numGeometries: UserDefinedFunction = udf(ST_NumGeometries)
+  def st_numPoints: UserDefinedFunction = udf(ST_NumPoints)
+  def st_pointN: UserDefinedFunction = udf(ST_PointN)
+  def st_x: UserDefinedFunction = udf(ST_X)
+  def st_y: UserDefinedFunction = udf(ST_Y)
+
+  /* Geometric Cast Functions */
+  def st_castToPoint: UserDefinedFunction = udf(ST_CastToPoint)
+  def st_castToPolygon: UserDefinedFunction = udf(ST_CastToPolygon)
+  def st_castToLineString: UserDefinedFunction = udf(ST_CastToLineString)
+  def st_castToGeometry: UserDefinedFunction = udf(ST_CastToGeometry)
+  def st_byteArray: UserDefinedFunction = udf(ST_ByteArray)
+
+  /* Geometric Constructor Functions */
+  def st_geomFromGeoHash: UserDefinedFunction = udf(ST_GeomFromGeoHash)
+  def st_box2DFromGeoHash: UserDefinedFunction = udf(ST_GeomFromGeoHash)
```

Comment:
just double checking this is the right delegate function?

Reply:
seems like it is from the comment in functions.py

Reply:
yep, it's just an alias for the other function